src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,435 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload tests running identical queries as tsbs_query in a Genny workload. During
+  development we noticed significant performance differences between Genny and tsbs
+  workloads running very similar queries. This workload is being used for development
+  to investigate these runtime differences. The data is preloaded by the dsi configuration.
+  See 'configurations/test_control/test_control.tsbs_query.yml' for all the details. There are 20736000 
+  documents in the collection.
+
+Keywords:
+  - timeseries
+  - aggregate
+  - group
+  - sort
+
+GlobalDefaults:
+  Database: &database cpu
+  Collection: &collection point_data
+  maxPhases: &maxPhases 12
+  Repeat: &repeat 10
+
+# Create variables to mimic the randomness of tsbs_generate_queries.
+#  The min and max time of the pre-loaded tsbs dataset.
+MinDate:
+  &MinDate {
+    ^RandomDate: { min: "2016-01-01T00:00:00Z", max: "2016-01-24T23:59:50Z" },
+  }
+
+MaxDate:
+  &MaxDate { ^RandomDate: { min: *MinDate, max: "2016-01-24T23:59:50Z" } }
+
+HostName1: &HostName1
+  LoadConfig:
+    Path: "../../phases/query/GetHostNameArray.yml"
+    Key: GetHostNameArray
+    Parameters:
+      count: 1
+
+HostName8: &HostName8
+  LoadConfig:
+    Path: "../../phases/query/GetHostNameArray.yml"
+    Key: GetHostNameArray
+    Parameters:
+      count: 8
+
+Actors:
+  # The data is already loaded into the collection from the dsi configuration.
+  - Name: Quiesce
+    Type: QuiesceActor
+    Threads: 1
+    Database: *database
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+
+  - Name: single_groupby_1_1_1
+    Type: CrudActor
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $match:
+                        {
+                          "tags.hostname": { $in: *HostName1 },
```

Comment:
I see you've dropped the useless `"measurement":"cpu"` filter... While I totally agree it's not adding value to the query, it does mean that these queries are diverging from the ones in tsbs_query. We should at least call it out in the Description section.